src/connector/src/source/manager.rs

```diff
@@ -34,6 +34,9 @@ pub struct SourceColumnDesc {
     // `is_pk` is used to indicate whether the column is part of the primary key columns.
     pub is_pk: bool,
 
+    // `is_hidden_addition_col` is used to indicate whether the column is a hidden addition column.
```

Comment:
```suggestion
    /// `is_hidden_addition_col` is used to indicate whether the column is a hidden addition column.
```

Please use rustdoc syntax.